components/Aduan/DaftarIKP/index.vue

```diff
@@ -364,6 +358,26 @@ export default {
     },
     goToPageDetail (id) {
       this.$router.push(`/aduan/penginputan-ikp/detail-ikp/${id}`)
+    },
+    async getCount () {
+      const queryCount = { ...this.query }
+      queryCount.status = ''
```

Comment:
just merge this code together

```suggestion
      const queryCount = { ...this.query, status: '' }
```

Reply:
sudah saya perbaiki di comment sebelumnya ya bli @Ibwedagama